leankyr/Easy_Problems/Caesar_Cipher/caesarCipher.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3.5
+# -*- coding: utf-8 -*-
+
+import sys
+
+def caesarCipher(s, k):
+    k = k%26
+    # Caps2integers Mappin
+    Caps2integers = dict(zip("ABCDEFGHIJKLMNOPQRSTUVWXYZ",range(0,26)))
```

Comment:
Follow the [naming conventions of PEP8](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles)